portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,7 +49,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String newComment = request.getParameter("new-comment");
-    comments.add(newComment);
+    Entity commentEntity = new Entity("comment");
+    commentEntity.setProperty("message", newComment);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
You would want to do this in init since you don't want to create a datastore service every time.